demo/js/index.js

```diff
@@ -147,6 +146,51 @@ Raven.context(async () => {
   let __butterchurnOptions = null;
   let __initialWindowLayout = null;
   if (isButterchurnSupported()) {
+    const partialButterchurnOptions = {
+      importButterchurn: () => {
+        return import(/* webpackChunkName: "butterchurn-initial-dependencies" */
+        // @ts-ignore
+        "butterchurn");
+      },
+      importConvertPreset: () => {
+        return import(/* webpackChunkName: "milkdrop-preset-converter" */
+        // @ts-ignore
+        "milkdrop-preset-converter-aws");
+      },
+      presetConverterEndpoint:
+        "https://p2tpeb5v8b.execute-api.us-east-2.amazonaws.com/default/milkdropShaderConverter",
+      getPresets: async () => {
+        if ("URLSearchParams" in window) {
+          const params = new URLSearchParams(location.search);
+          const butterchurnPresetUrlParam = params.get("butterchurnPresetUrl");
+          const milkdropPresetUrl = params.get("milkdropPresetUrl");
+          const initialPresets = [];
+          if (butterchurnPresetUrlParam) {
+            initialPresets.push({
+              name: butterchurnPresetUrlParam.slice(
+                0,
+                butterchurnPresetUrlParam.length - ".json".length
+              ),
+              butterchurnPresetUrl: butterchurnPresetUrlParam
+            });
+          } else if (milkdropPresetUrl) {
+            throw new Error("We still need to implement this");
+          }
+          const resp = await fetch(
+            "https://unpkg.com/butterchurn-presets-weekly@0.0.2/weeks/week1/presets.json"
```

Comment:
@jberg Were you thinking of setting up an endpoint which would redirect to the current week's presets?